src/Clock.tsx

```diff
@@ -0,0 +1,65 @@
+import { Component } from 'react';
+
+type State = {
+  today: Date;
+};
+
+type Props = {
+  clockName: string;
+};
+
+const formatTime = (time: Date) => {
+  return time.toUTCString().slice(-12, -4);
+};
```

Comment:
on next time move all helper functions out of component smth there src/helpers/name.ts